SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -391,12 +391,10 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	}
 
 	void SetPrimaryLaunchConfigPath(const FString& Path);
-	FORCEINLINE FString GetPrimaryLanchConfigPath() const
+	FORCEINLINE FString GetPrimaryLaunchConfigPath() const
```

Comment:
GetPrimary**Lanch**ConfigPath

did this typo break some functionality previously?

Reply:
Nope, it was just misspelled in all places